src/main/java/com/google/sps/data/DatastoreClientInterface.java

```diff
@@ -17,15 +17,15 @@
   // If object already exist, the old one is replaced. 
   void insertOrUpdateSession(SessionInterface session);
    // Returns the Session object associated with the given sessionId.
-  Optional<Session> getSession(String sessionId) throws Exception;
+  Optional<Session> getSession(String sessionId);
   // Returns the Instance object associated with the given instanceName.
-  Optional<Instance> getInstance(String instanceName) throws Exception;
+  Optional<Instance> getInstance(String instanceName);
   // Returns the Attendee object associated with the given screenName.
-  Optional<Attendee> getAttendee(String screenName) throws Exception;
+  Optional<Attendee> getAttendee(String screenName);
```

Comment:
Please add a comment for each of these get functions that if the return optional is empty, then no object with the associated parameter does not exist.

Reply:
Also, please adhere to the javadoc style of writing interface comments. 

https://google.github.io/styleguide/javaguide.html#s7-javadoc